crt/exit.c

```diff
@@ -24,9 +25,13 @@ void exit(int code)
 {
     bool am_parent_of_fork = false;
     bool am_child_fork = false;
+    long fork_mode = 0;
 
-    if (syscall(SYS_myst_is_shared_crt, &am_parent_of_fork, &am_child_fork) ==
-        0)
+    if (syscall(
+            SYS_myst_get_fork_mode,
+            &fork_mode,
```

Comment:
This should be ``myst_fork_mode_t`` pointer because ``sizeof(myst_fork_mode_t) != sizeof(long)``

Reply:
This is a pointer error.

Reply:
fixed